pkg/minishift/hostfolder/sshfs_hostfolder.go

```diff
@@ -102,7 +102,7 @@ func (h *SSHFSHostFolder) Mount(driver drivers.Driver) error {
 }
 
 func (h *SSHFSHostFolder) Umount(driver drivers.Driver) error {
-	cmd := fmt.Sprintf("sudo umount %s", h.config.MountPoint())
+	cmd := fmt.Sprintf("sudo fusermount -u %s", h.config.MountPoint())
```

Comment:
@anjannath What is the advantage of using `fusermount` instead `umount` also does that command available on each iso which we have current support?

Reply:
I am still looking for the advantages of it over umount, just that it was listed on the man page for [SSHFS](https://linux.die.net/man/1/sshfs). It is available in the centos iso and b2d, moreover it is user space cli for the FUSE file system so it should be present on all the isos.

Reply:
Can you verify that this command is available? I would expect so, as it is part of our ISO... however, we change something that works and is consistent between all other implementations? Without any benefit, i would not see a need to change this.

Reply:
@gbraad Yes i have verified it is available in all b2d, centos iso, i haven't checked in the fedora iso though..

Reply:
Also i think, umount internally calls `fusermount -u` so this might not be necessary.. but since it was mentioned on the SSHFS man page i decided to go with it, we can change back to `umount` as that was also working fine.

Reply:
well, for mount in that case we could also use `mount.xfs`, `mount,fuse`, etc... but when the abstraction makes sense, please use it. `umount` works for everything as expected; let the abstraction do its' work.

Reply:
@gbraad Okay.. reverting this change.